src/App.tsx

```diff
@@ -1,61 +1,161 @@
+/* eslint-disable default-case */
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userArgument: number | string): User | null {
+  const foundUserById
+  = usersFromServer.find(user => user.id === userArgument);
+
+  const foundUserByName
+  = usersFromServer.find(user => user.name === userArgument);
```

Comment:
what is `userArgument`?
it seems that this is an object and not a name or ID
rename the variables so that it is clear from the code what type the variable has